src/status_im/utils/signing_phrase/core.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.utils.signing-phrase.core
+  (:require [status-im.utils.signing-phrase.dictionaries.en :as en]
+            [status-im.utils.signing-phrase.dictionaries.ru :as ru]
+            [clojure.string :as string]))
+
```

Comment:
Could you add a short paragraph here about the purpose and guarantees of signing phrase? Also, what does the process look like for other locales going forward? What happens if you have a different locale?

Reply:
Re docs: Good idea

Re languages: Actually, that's an open question - I created a separate issue for the language question (#1679) so that this one would be well scoped. My main issue with it is that device language is stored independently of our db so we could have a case where language changes and signing phrase remains in the previous one. It might be a non issue if we allow the user to re-generate signing phrase on demand.